lib/style/module_directives.ex

```diff
@@ -333,8 +333,8 @@ defmodule Styler.Style.ModuleDirectives do
 
     if Enum.any?(liftable) do
       # This is a silly hack that helps comments stay put.
-      # the `cap_line` algo was designed to handle high-line stuff moving up into low line territory, so we set our
-      # new node to have an abritrarily high line annnnd comments behave! i think.
+      # The `cap_line` algorithm was designed to handle high-line stuff moving up into low line territory,
+      # so we set our new node to have an arbitrarily high line so comments stay where they are.
```

Comment:
i'm happy to take typo fixes, but prefer my comments to myself remaining my comments to myself in tone and style, including nonsensical things like "annnnnd"